R/GitHost.R

```diff
@@ -63,9 +63,18 @@ GitHost <- R6::R6Class("GitHost",
     },
 
     # Get repositories URLS from the Git host
-    get_repos_urls = function(type = "web", file = NULL, verbose, settings) {
+    get_repos_urls = function(type = "web", code = NULL, file = NULL, verbose, settings) {
```

Comment:
I know this is a method, but wouldn't be better if parameters `file` & `code` were the same as in exported functions i.e. `with_file` & `with_code`?

Reply:
I'll address this issue in next PR.